EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1230,273 +1230,273 @@ QUEST_LV_0300_20160310_001229	Even if you can't trust me you should trust Vaidot
+QUEST_LV_0300_20160310_001250	I felt so painful since I didn't know what to do.{nl}Ah... The abbot!{nl}
+QUEST_LV_0300_20160310_001251	The abbot must be locked inside the monastery.{nl}I was hoping that the goddesses would protect us... {nl}But we will be in trouble if we don't quickly give you the food.
+QUEST_LV_0300_20160310_001252	Elgos Abbot
+QUEST_LV_0300_20160310_001253	This energy... Yes, it's you. The Revelator.{nl}It seems the goddess finally answered my prayer.{nl}
+QUEST_LV_0300_20160310_001254	Ah, nothing. If the other monks are alright, the food will be okay...{nl}I don't think I will be able to swallow the food.
+QUEST_LV_0300_20160310_001255	He is too old... Is he going to be alright...?
+QUEST_LV_0300_20160310_001256	You are not blocked by the black wall since you are the Revelator, as expected.
+QUEST_LV_0300_20160310_001257	The evil idol that was created by Dominikas in front of Pagarba Chapel...{nl}The black wall started to cover the monastery because of that idol.{nl}
+QUEST_LV_0300_20160310_001258	If we destroy that evil idol...{nl}There might be a good chance that the whole wall may disappear.
+QUEST_LV_0300_20160310_001259	I just don't know how I can destroy it...{nl}There is nothing I can think of at the moment.{nl}
+QUEST_LV_0300_20160310_001260	But, wait...{nl}I can sense the soul of the demons in you.{nl}
+QUEST_LV_0300_20160310_001261	That sphere you have...{nl}Is a demon's soul locked inside it?{nl}
+QUEST_LV_0300_20160310_001262	That's it.{nl}Maybe we can use this to destroy the evil idol.{nl}
+QUEST_LV_0300_20160310_001263	The idea is to purify the soul of the demons and use their power.{nl}Unfortunately, we lack the right amount of souls to destroy the idol.{nl}
+QUEST_LV_0300_20160310_001264	Can you fill the demon's soul a little more?{nl}As you already may know, defeating demons will automatically absorb their souls.
+QUEST_LV_0300_20160310_001265	The demons are strong so... I guess that their souls are strong as well.{nl}My sacred energy is not enough with this much power.
+QUEST_LV_0300_20160310_001266	We should somehow borrow the power of holy water...{nl}Like I told you before though, I have nothing at the moment.
+QUEST_LV_0300_20160310_001267	I am so sorry, but can you get me Holy Water from the Priest Master in Klaipeda?{nl}If we have enough holy water, we'll be able to purify the collapsed demon souls.
+QUEST_LV_0300_20160310_001268	Please use the statue of Goddess Vakarine near Viyme Fountain.{nl}You can quickly come back from Klaipeda.
+QUEST_LV_0300_20160310_001269	With this amount, you will be able to purify the Soul Sphere.{nl}Now, give me a moment.
+QUEST_LV_0300_20160310_001270	Since the power of a demon's soul is strong, the purified power is strong as well.{nl}This power has the capability to destroy the evil idol.{nl}
+QUEST_LV_0300_20160310_001271	Put the Soul Sphere's power into the purified evil idol.{nl}I am not sure if the idol is able to endure...
```

Comment:
I think the word order is messed up here.  There no reason for the idol to be purified at this point.  Should it be:

"Use the Soul Sphere's purified power on the evil idol.{nl}I hope the idol can't endure this..."